app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -214,7 +236,7 @@ describe('League entries page (Entry Component)', () => {
     expect(viewPastWeeksLink).toBeInTheDocument();
   });
 
-  it('should not display a button to add a new entry if there are more than 5 entries', async () => {
+  it('should not display a button to add a new entry if there are 5 entries', async () => {
```

Comment:
But this expected behavior also applies when there are more than 5 entries too, right? So maybe we could reword this to: `should not display a button to add a new entry if there are 5 or more entries`

Reply:
nope.  because there will never be more than 5 entries.